SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialLatencyTracer.h

```diff
@@ -139,6 +143,7 @@ class SPATIALGDK_API USpatialLatencyTracer : public UObject
 	void ClearTrackingInformation();
 
 	FString WorkerId;
+	static FString MessagePrefix;
```

Comment:
No need for static here, we typically get an instance of the tracer in the static functions and call a member func

Reply:
We discussed this offline, but for the record: The latency tracer singleton may be unitialized at the start of the game. This would mean that any static method that attempts to update a local variable could fail if called too early. Rather than have this risk, I chose to make this field static.

Reply:
Could we add a comment saying that RegisterProject always succeeds so we have to ensure that we can store the parameters?

Reply:
I've added a comment, please TAL